demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
I think I agree with @triblondon's rational, while those that understand this method for doing grids, others might not and it might look like a bug. With the goal of getting people started with components more easily I think this works. Those that need to debug can easily add a row colour through the inspector or editing the SCSS themselves.